_wiki/FAQ/General.md

```diff
@@ -2,63 +2,112 @@
 title: General FAQ
 ---
 
-### Application Data Directory {#appdata}
```

Comment:
I don't think we should be dropping predefined IDs. They're handy for allowing us to permalink to a heading from outside the site (e.g. Discord links) even if the title changes.
(same goes for other defined ID that are dropped)